course-definition.yml

```
@@ -439,41 +439,67 @@ stages:
     name: "RDB file config"
     difficulty: easy
     description_md: |
-      Redis uses `.rdb` files for persistence. In this stage, you'll add support for reading the config values related to where RDB files are stored.
+      Welcome to the RDB Persistence Extension! In this extension, you'll add support for reading [RDB files](https://redis.io/docs/management/persistence/) (Redis Database files).
 
-      There are two config values that determine where RDB files are stored:
+      In this stage, you'll add support for two configuration parameters related to RDB persistence, as well as a command that returns the values of configuration parameters.
```

```suggestion
      In this stage, you'll add support for two configuration parameters related to RDB persistence, as well as the [CONFIG GET](https://redis.io/docs/latest/commands/config-get/) command.
```

(Shorter + adds some "mystery" to the hook)